### PR TITLE
Query and Update DB

### PR DESCRIPTION
**Pull request by Habtamu210**
### Project includes:
- Add a column species of type string to the animal's table. 
- Modify your schema file.
- Insert data
- Inside a transaction update the animals' table by setting the species column to unspecified
- Take a screenshot of the results of actions.
-  Inside a transaction delete all records in the animal's table, then roll back the transaction.
- After the rollback verify if all records in the animal's table still exist. After that, you can start breathing as usual 
- Write queries to answer the questions listed

Capture 001
Capture00
Capture01
Capture0002
Capture002
Capture02
Capture004
Capture04
Capture005
Capture05
Capture06

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,18 +1,46 @@
...
+SELECT * FROM animals
+
+ROLLBACK;
+SELECT * FROM animals
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals
```
Comment (AnselemOdims) [inline queries.sql]: - [ ] Please kindly verify the change was made before commiting

File: queries.sql [hunk 2]
```diff
@@ -1,18 +1,46 @@
...
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals
```
Comment (AnselemOdims) [inline queries.sql]: - [ ] Please kindly verify the delete change was made before rolling back

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @Habtamu210 , 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by TadesseDev — APPROVED** [review]
### Hi @Habtamu210 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**